draft-ietf-tls-esni.md

```diff
@@ -767,7 +776,9 @@ messages, but just blindly forwards them.
 
 If the server sends a NewSessionTicket message, the corresponding ECH PSK MUST
```

Comment:
Perhaps this can say that servers MUST NOT use this PSK if ECH is rejected?

Reply:
That's ensured by the rule change above, correct? If so, then the server won't send a NewSessionTicket message.

Reply:
Yep!

Reply:
I dropped the TODO and left the text as-is for now.